src/App.css

```diff
@@ -61,6 +61,44 @@
    vertical-align: middle;
 }
 
+.delete-button {
+  margin-left: 10px;
+  width:  48px;
+  height: 48px; 
+  display: flex;
+  border-radius: 10px;
+}
+
+.new-task-container{
+  margin-top: 50px;
+  display: flex;
+  width: 770px;
+  height: 67px;
```

Comment:
There shouldn't be a static `height` and `width` for this element. Its `width` should stretched to be whatever the task elements' width is and its `height` should be determined by its child element